index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Eastern Designs</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="style.css" rel="stylesheet"/>
+  <link href="navbar.css" rel="stylesheet"/>
+  <script src="https://kit.fontawesome.com/a012cea4ea.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+<header class="header">
+  <nav class="navbar">
+      <a href="#" class="nav-logo">WebDev.</a>
+      <ul class="nav-menu">
+          <li class="nav-item">
+              <a href="#" class="nav-link">Services</a>
+          </li>
+          <li class="nav-item">
+              <a href="#" class="nav-link">Blog</a>
+          </li>
+          <li class="nav-item">
+              <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+              <a href="#" class="nav-link">Contact</a>
+          </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </nav>
+</header>
+<div>

```

Comment:
- [ ] Please use the semantic tag like `<section>` in place of the generic `<div>` tag, this will ensure your page is optimized for SEO, and your page is easily readable.